### ISSUE TITLE
aria-live attribute on Dropdown doesn't apply to child span 

### ISSUE THREAD

**Issue by jemeese** [microsoft/fluentui#19346]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@7.137.2
- **Browser and OS versions**: Edge, os.ver 21390.2025

### Describe the issue:

Setting aria-live on a Dropdown has no effect, as the attribute is not inherited by the span which displays the selected dropdown option.  Setting it like so:

    <Dropdown aria-live="off" options={options} >
    </Dropdown>
Results in: 
![image](https://user-images.githubusercontent.com/58831720/128907107-2c4fca34-d5dd-41dd-8ee0-32f44fcf23bf.png)

This is an accessibility issue as we have times where we cascade auto-populate dropdowns in cases of their being only one option, causing Narrator to read out every single dropdown we fill (without actually naming the label of the dropdown being populated, so it is not useful to the user).

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/jemeese/pen/OJmrGpy

#### Actual behavior:

aria-live="polite" is set on the span of the dropdown

#### Expected behavior:

aria-live="off" or no attribute is set on the span of the dropdown

### Documentation describing expected behavior

https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B64DD7C5C-AA62-4D53-886E-BCAEE5341C01%7D&file=Guideline%20101%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties_v3_(CELA).docx&action=default&mobileredirect=true

HCL accessibility team filed us issue on us with the expected result of "Screen reader should announce only correct or relevant information".

**Comment by jemeese** [comment]
I've tried onRenderOption, onRenderItem, and onRenderTitle, and none of them seem to give access to change the span element which sets aria-live="polite" (the selected item in a dropbox).  I might just be missing something though.

**Comment by gouttierre** [comment]
@jemeese - Thanks for filing this issue with us and providing the details into the problem you are facing.
  
@bsunderhus - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by smhigley** [comment]
@jemeese we implemented a fix for this in #19667, where the `aria-live` attributes are only added when the Dropdown is in focus. It was ported to v7 in this PR: #23425

If you update your Fluent version to any v7 version after June 2022, you should get the fix, as well as a number of other critical accessibility fixes (e.g. I see you still have `role="listbox"` on there...) 😁
